src/api/app/views/models/_project.xml.builder

```diff
@@ -41,13 +41,13 @@ xml.project(project_attributes) do
       repo.download_repositories.each do |download_repository|
         params = {arch: download_repository.arch, url: download_repository.url, repotype: download_repository.repotype}
         xml_repository.download(params) do |xml_download|
-          xml_download.archfilter download_repository.archfilter if download_repository.archfilter
+          xml_download.archfilter download_repository.archfilter if !download_repository.archfilter.blank?
```

Comment:
IMHO we shouldn't have empty strings in the DB, there might be a problem elsewhere when this is needed.